### ISSUE TITLE
Authentication problem with solr

### ISSUE THREAD

**Issue by zelima** [datopian/ckan-cloud-operator#19]
Irakli Mchedlishvili [Today at 1:48 PM]
Getting this after migrating the first instance on the production
```
Traceback (most recent call last):
  File "/usr/bin/paster", line 11, in <module>
    sys.exit(run())
...
  File "/usr/lib/python2.7/httplib.py", line 736, in __init__
    (self.host, self.port) = self._get_hostport(host, port)
  File "/usr/lib/python2.7/httplib.py", line 777, in _get_hostport
    raise InvalidURL("nonnumeric port: '%s'" % host[i+1:])
httplib.InvalidURL: nonnumeric port: '***@***.searchstax.com'
```

we need to set:
```
CKAN_SOLR_USER
CKAN_SOLR_PASSWORD
```

the change should be done here:
https://github.com/ViderumGlobal/ckan-cloud-operator/blob/master/ckan_cloud_operator/deis_ckan/envvars.py#L49 (edited)

and in infra secret - add the username / password as well in separate keys

**Comment by zelima** [comment]
Above was solved by adding CKAN_SOLR_USER and CKAN_SOLR_PASSWORD to envvars see https://github.com/ViderumGlobal/ckan-cloud-operator/pull/20

But now we have authentication problem anyway when making requests:
```
Error: SolrError(u'Solr responded with an error (HTTP 401): [Reason: None]\n<html><head><meta http-equiv="Content-Type" content="text/html; charset=UTF-8"/><title>Error 401 Unauthorized request, Response code: 401</title></head><body><h2>HTTP ERROR 401</h2><p>Problem accessing /solr/lincolnshire/select/. Reason:<pre>    Unauthorized request, Response code: 401</pre></p><hr><i><small>Powered by Jetty://</small></i><hr/></body></html>',)
```

**Comment by zelima** [comment]
FIXED in ViderumGlobal/ckan-cloud-operator@afde17b
